assets/app.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Welcome to your app's main JavaScript file!
+ *
+ * We recommend including the built version of this JavaScript file
+ * (and its CSS file) in your base layout (base.html.twig).
+ */
+
+// any CSS you import will output into a single css file (app.css in this case)
+import './styles/app.css';
+
+// start the Stimulus application
+import './bootstrap';
+
+var $ = require( "jquery" );
+
+const exit = e => {
+    const shouldExit =
+        [...e.target.classList].includes('exit-intent-popup') || // user clicks on mask
+        e.target.className === 'close' || // user clicks on the close icon
+        e.keyCode === 27; // user hits escape
+
+    if (shouldExit) {
+        document.querySelector('.exit-intent-popup').classList.remove('visible');
+    }
+};
+
+const mouseEvent = e => {
+    const shouldShowExitIntent =
+        !e.toElement &&
+        !e.relatedTarget &&
+        e.clientY < 10;
+
+    if (shouldShowExitIntent) {
+        document.removeEventListener('mouseout', mouseEvent);
+        document.querySelector('.exit-intent-popup').classList.add('visible');
+
+
+    }
+};
+
+document.querySelector('.exit-intent-popup').addEventListener('click', exit);
+
+setTimeout(() => {
+    document.addEventListener('mouseout', mouseEvent);
+    document.addEventListener('keydown', exit);
+}, 5_000);
+
+$(document).ready(function(){
+    console.log("jQuery working properly");
+});
+
+$(".slideButton1").click(function(){
```

Comment:
klases kurias naudoji js'ui, stenkis pavadineti js-_klases_name, jeigu tavo objektas turi jau klase kazkokia, sukurt nauja js'ui, nelipink prie esancios. Patogiau skaitysis ir bus aisku is karto, kas ir kaip.